spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/logging/NacosLoggingAppRunListener.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2013-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.nacos.logging;
+
+import com.alibaba.nacos.client.logging.NacosLogging;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.SpringApplicationRunListener;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.Ordered;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+/**
+ * Reload nacos log configuration on spring application contextPrepared.
+ *
+ * @author RuanSheng
+ */
+public class NacosLoggingAppRunListener implements SpringApplicationRunListener, Ordered {
+
+	public NacosLoggingAppRunListener(SpringApplication application, String[] args) {
+
+	}
+
+	@Override
+	public void environmentPrepared(ConfigurableEnvironment environment) {
+		NacosLogging.getInstance().loadConfiguration();
```

Comment:
Whether we don't need do load configuration here? because I see it doesn't have clearing logContext logic before here and just have it before contextPrepared like following:
![image](https://user-images.githubusercontent.com/25423978/220317426-9277a951-59a0-4963-afe8-766b294a6338.png)

Reply:
The Nacos client will create connection after `environmentPrepared`, and log messages will be output to the console. If the `NacosLogging.getInstance().loadConfiguration()` is reset after `contextPrepared`, some of the log messages will be output to the console between `environmentPrepared` and `contextPrepared`.